### PR TITLE
README changed to Markdown as suggested in issue #8

### PR DESCRIPTION
**Pull request by cdwensley**
Just trying out the system.  
Not at all sure about the first four lines. 
What to do about the copyright material in makedoc.g?

### CODE REVIEW

**Comment by alex-konovalov** [comment]
@cdwensley github rendering looks almost OK: https://github.com/cdwensley/gbnp/tree/readme

What exactly makes you unsure about the first lines?

**Comment by cdwensley** [comment]
I have not the slightest idea about this syntax.  Just copied some lines from (I think) Utils; changed the package name to gbnp, and hoped for the best.

**Comment by alex-konovalov** [comment]
@cdwensley since GBNP already has CI setup in place, clicking on badges at https://github.com/cdwensley/gbnp/tree/readme navigates to the right place, so it works.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+ * From Version 0.9 on the package is enriched with support for GAP fields and
+   additional prefix rules for quotient modules as well as some speed
+   improvements by Jan Willem Knopper. Knopper has also formatted the
+   documentation in GAPDoc.
+
+ * From Version 1.0 on the package is extended with NMO (for Noncommutative
+   Monomial Orderings) by Randall Cone. This enables the GBNP user to choose a
+   wider selection of monomial orderings than the standard one built into GBNP
+   itself. The files of this extension can be found in doc/nmo,
```
Comment (alex-konovalov) [inline README.md]: add `...` to paths

File: README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Distribution
+
+ * The GitHub package repository is at 
+   <https://github.com/gap-packages/gbnp/> 
+ * and the latest release is available at 
+   <https://gap-packages.github.io/gbnp/>
+
+## Copyright
+
+The 'GBNP' package is Copyright {\copyright} The GAP Group, 2001-2020.
```
Comment (alex-konovalov) [inline README.md]: Do we need `{\copyright}`?

**Comment by cdwensley** [comment]
This all came about because I have started work on a package IBNP based on GBNP and had a conversation with Max on Slack last Monday, asking who best to talk to about possible changes to GBNP.  I gather that discussions with the original authors are ongoing.  So I thought I should offer to help with some of the Github issues and maybe some items in the TODO list.

File: README.md (hunk 1 above)
Comment (cdwensley) [reply README.md]: You mean .../gbnp/doc/nmo etc.?

File: README.md (hunk 2 above)
Comment (cdwensley) [reply README.md]: Seems unlikely.  Again I just copied this from Utils.  Not sure whether I put it there or Max.

File: README.md (hunk 1 above)
Comment (alex-konovalov) [reply README.md]: yes

File: README.md [hunk 3]
```diff
@@ -51,10 +51,10 @@ of the GAP root directory.
 
  * The theory is mainly taken from literature by Teo Mora and Edward L. Green.
 
- * From Version 0.8.3 on the package has three additional files (fincheck.g,
-   tree.g graphs.g) with routines for finding the Hilbert function and testing
-   finite-dimensionality when given a GB by Chris Krook, based on work by
-   Victor Ufnarovski.
+ * From Version 0.8.3 on the package has three additional files by Chris Krook, 
+   based on work by Victor Ufnarovski. These files (fincheck.g, tree.g,
```
Comment (alex-konovalov) [inline README.md]: Same here - I suggest to put filenames in backticks `...`

Comment (cdwensley) [reply README.md]: done

**Comment by cdwensley** [comment]
Should a squash now be attempted, or is another review required?

**Comment by alex-konovalov** [comment]
squash is fine!

**Comment by cdwensley** [comment]
@alex-konovalov  At last I think I understand how to squash!  But now a GitHub problem: 
```ChristoleysiMac:gbnp christopherwensley$ git rebase -i master
[detached HEAD 5c6e46d] README changed to Markdown as suggested in issue #8
 Date: Mon Jan 6 17:43:14 2020 +0000
 2 files changed, 83 insertions(+), 56 deletions(-)
 delete mode 100644 README
 create mode 100644 README.md
Successfully rebased and updated refs/heads/readme.
ChristoleysiMac:gbnp christopherwensley$ git push origin readme
To https://github.com/cdwensley/gbnp
 ! [rejected]        readme -> readme (non-fast-forward)
error: failed to push some refs to 'https://github.com/cdwensley/gbnp'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'''

Should I now:  "git push --force-with-lease origin readme", or what?

**Comment by alex-konovalov** [comment]
Yes, if you rebased your branch locally, you need to force push in this case.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+[![Build Status](https://travis-ci.org/gap-packages/gbnp.svg?branch=master)](https://travis-ci.org/gap-packages/gbnp)
```
Comment (fingolfin) [inline README.md:1]:
```suggestion
[![Build Status](https://github.com/gap-packages/gbnp/workflows/CI/badge.svg?branch=master)](https://github.com/gap-packages/gbnp/actions?query=workflow%3ACI+branch%3Amaster)
```

**Review by fingolfin — APPROVED** [review]
Thank you!

**Pull request merged**